src/lib.rs

```diff
@@ -621,6 +622,7 @@ fn rdo_mode_decision(fi: &FrameInvariants, fs: &mut FrameState,
         if fi.frame_type == FrameType::KEY && mode >= PredictionMode::NEARESTMV {
           break;
         }
+
```

Comment:
and here?

Reply:
This line break was added for readability.